### PR TITLE
Update README.md based on changes in EOS-16512

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**

### CODE REVIEW

**Comment by SwapnilGaonkar7** [comment]
@mssawant @knekrasov Please have a look

**Comment by knekrasov** [comment]
Please reword your commit and resolve DCO issue.

**Comment by SwapnilGaonkar7** [comment]
> Please reword your commit and resolve DCO issue.

@knekrasov Done

**Comment by knekrasov** [comment]
Suggestion for the commit message

```
Problem:  some sections in RFC 20 are incomplete

Solution: add more details to the sections describing implementation details of post_install and init commands.
```

(Problem line is the commit header, Solution (and probably some more context) go into the commit message)

I know that for this little change this comment may sound like nitpicking but this is a general rule how the commits should look like in Hare project.

File: rfc/20/README.md [hunk 1]
```diff
@@ -115,10 +115,11 @@ TBD
 
 #### post_install
 ```
-setup_hare post_install --config <str>
+hare_setup.py --post_install
```
Comment (knekrasov) [inline rfc/20/README.md:118]: Now `hare_setup.py` is obsolete. Now it is indeed `hare_setup` executable.

Comment (SwapnilGaonkar7) [reply rfc/20/README.md]: Done

**Comment by SwapnilGaonkar7** [comment]
@knekrasov @mssawant  Please have a look

**Review by knekrasov — APPROVED** [review]

**Pull request merged**